### ISSUE TITLE
[Final 0.12 issue] Translations need to be updated for release

### ISSUE THREAD

**Issue by LtxProgrammer** [LtxProgrammer/Changed-Minecraft-Mod#498]
**TODO**
- es_es.json
- fi_fi.json
- fr_fr.json
- ja_jp.json
- ko_kr.json
- no_no.json
- ru_ru.json
- sv_se.json
- zh_cn.json

**Comment by Foxyas** [comment]
can i send a pt_br translation of this mod in a pull request if yes how i make a pull request?

**Comment by LtxProgrammer** [comment]
> can i send a pt_br translation for the mod in a pull request if yes how i make a pull request?

Yes you can. And read here for pull requests https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/getting-started/about-collaborative-development-models

**Comment by Foxyas** [comment]
> > can i send a pt_br translation for the mod in a pull request if yes how i make a pull request?
> 
> Yes you can. And read here for pull requests https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/getting-started/about-collaborative-development-models

I just made a pull request, if there's something wrong please let me know

**Comment by BecoolTJ** [comment]
whats the current completion status of the remaining translations? if they havent been worked on yet / have barely been worked on i think releasing a patch when they are completed is reasonable to shorten the wait time for 0.12.
